libp2p/standard_setup.nim

```diff
@@ -1,71 +0,0 @@
-import
```

Comment:
could leave an import/export in place which would allow existing code not to break

Reply:
Yeah, good point and I ran into it in nimbus, will add back to avoid breaking other projects.